app/src/main/java/org/thoughtcrime/securesms/database/model/ThreadRecord.java

```diff
@@ -144,14 +144,17 @@ public CharSequence getDisplayBody(@NonNull Context context) {
                     .format().toString();
 
         } else if (MmsSmsColumns.Types.isMessageRequestResponse(type)) {
-            if (lastMessage.getRecipient().getAddress().serialize().equals(
-                    TextSecurePreferences.getLocalNumber(context))) {
-                return UtilKt.getSubbedCharSequence(
-                        context,
-                        R.string.messageRequestYouHaveAccepted,
-                        new Pair<>(NAME_KEY, getName())
-                );
+            try {
+                if (lastMessage.getRecipient().getAddress().serialize().equals(
+                        TextSecurePreferences.getLocalNumber(context))) {
+                    return UtilKt.getSubbedCharSequence(
+                            context,
+                            R.string.messageRequestYouHaveAccepted,
+                            new Pair<>(NAME_KEY, getName())
+                    );
+                }
             }
+            catch (Exception e){} // the above can throw a null exception
```

Comment:
Should we address the NPE rather than catching it? Like `getName() != null` as part of the condition

Reply:
@SessionHero01 Ideally yes but I couldn't be sure which part was returning null here. 
Are we sure it is `getName()`?
The store seem to think it was here:
```
lastMessage.getRecipient().getAddress().serialize().equals(
                        TextSecurePreferences.getLocalNumber(context)))
```

Reply:
You are right there's no clear indication which one is null